### PR TITLE
Make yuks retry 10 times

### PR DESCRIPTION
**Pull request by krzyzacy**
yuks should try harder!

ohh hope I can get something other than ground beef

https://github.com/kubernetes/test-infra/pull/3495#issuecomment-315230198

### CODE REVIEW

**Comment by spxtr** [comment]
Please fix :)

**Comment by krzyzacy** [comment]
wow it cannot end with punctuation 😢

File: prow/plugins/yuks/yuks.go [hunk 1]
```diff
@@ -98,13 +99,18 @@ func handle(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent, j j
...
+			log.Infof("Commenting with \"%s\".", resp)
+			return gc.CreateComment(org, repo, number, plugins.FormatICResponse(ic.Comment, resp))
+		}
+
+		log.Errorf("joke contains invalid characters: %v", resp)
 	}
-	log.Infof("Commenting with \"%s\".", resp)
-	return gc.CreateComment(org, repo, number, plugins.FormatICResponse(ic.Comment, resp))
+
+	return errors.New("all 10 jokes contain invalid character... such an unlucky day")
```
Comment (fejta) [inline prow/plugins/yuks/yuks.go]: lol

**Pull request merged**